db/db.go

```diff
@@ -846,6 +851,26 @@ func UpdateConnectionsOfDeletedUser(userId int, tenantName string) error {
 	return nil
 }
 
+func RemoveConnectionsOfDeletedUsers(tenantNames string) error {
+	ctx, cancelfunc := context.WithTimeout(context.Background(), DbOperationTimeout*time.Second)
+	defer cancelfunc()
+	conn, err := MetadataDbClient.Client.Acquire(ctx)
+	if err != nil {
+		return err
+	}
+	defer conn.Release()
+	query := `DELETE FROM connections WHERE tenant_name=$1`
```

Comment:
this will delete all connections of tenant not user, is that what you meant for?